src/webgpu/api/operation/command_buffer/image_copy.spec.ts

```diff
@@ -625,6 +626,149 @@ class ImageCopyTest extends GPUTest {
         unreachable();
     }
   }
+
+  async CheckStencilTextureContent(
```

Comment:
unit: checkStencilTextureContent

Reply:
FYI @kainino0x maybe we want to move this texture helper to be shared between tests in the future.

Might be worth doing a perfect copy of the stencil data instead of checking each pixel individually below. It should be possible to do this with 8 draws with blendMode ADD that each check a single bit of the stencil buffer.

Reply:
As a crazy hack we could use the instanceIndex to pass extra data with baseInstance without adding a uniform buffer ^^

Reply:
> Might be worth doing a perfect copy of the stencil data instead of checking each pixel individually below. It should be possible to do this with 8 draws with blendMode ADD that each check a single bit of the stencil buffer.

This is one of the things I want to have a texture helper for.

Reply:
As in WebGPU we support copying from the stencil aspect into a buffer, I think it easier to verify the stencil data with T2B copies. In this test I am testing the B2T and T2B copies, so I have to check the stencil data in a render pass, so it seems we just need to keep this code here.

Reply:
> Might be worth doing a perfect copy of the stencil data instead of checking each pixel individually below. It should be possible to do this with 8 draws with blendMode ADD that each check a single bit of the stencil buffer.

Do you mean using 8 render pipelines with different stencilReadMask (1 << 0 to 1<< 7), and in each draw we output ( (1<<0) / 255, (1 << 1) / 255, ...  (1 << 7) / 255) to an R8Unorm color output?

Reply:
Yes exactly.